frontend/src/helpers/toConstantCase.js

```diff
@@ -0,0 +1,5 @@
+import { upperCase } from 'lodash'
+
+export const toConstantCase = (str) => {
+  return upperCase(str).replace(/ /g, '_')
```

Comment:
Let's not create a dependency on Lodash for this. If we change this to `str.toUpperCase().replace(/ /g, '_')`, it'll give us the same behaviour without dragging in new a dependency.

Reply:
Done 👍 

```
export const toConstantCase = (str) => {
  return str.toUpperCase().replace(/ /g, '_')
}

```